src/tools/bin/tbg

```diff
@@ -430,7 +430,7 @@ if [ -d "$job_outDir" ] ; then
     if [ "$force" == "true" ] ; then
         echo "Warning: using existing folder on user-request [-f]"
     else
-        echo "destination path already in use, can't create new folder"
+        echo "Destination path already in use, can't create new folder" >&2
```

Comment:
OPTIONAL: 
If you want a capital letter to start a sentence please ad a dot in the end too. 
Good point with streaming this to stderr.

Reply:
regarding @jkelling reply below: I can accept that - I will merge as soon as the compile test is possitive

Reply:
please write `cannot` while we are on it.